### PR TITLE
Modified get_retirement_endpoint to set default for Ansible playbook.

### PR DESCRIPTION
**Pull request by billfitzgerald0120**
Set to new generic statemachine if blank for service type ServiceAnsiblePlaybook.

### CODE REVIEW

**Comment by billfitzgerald0120** [comment]
@miq-bot add-label euwe/no

File: content/automate/ManageIQ/Service/Retirement/StateMachines/Methods.class/__methods__/get_retirement_entrypoint.rb [hunk 1]
```diff
@@ -5,11 +5,16 @@
...
 entry_point = service.automate_retirement_entrypoint
+$evm.log("info", "Starting get_retirement_entrypoint: #{entry_point}")
 if entry_point.blank?
-  entry_point = '/Service/Retirement/StateMachines/ServiceRetirement/Default'
-  $evm.log("info", "retirement_entrypoint not specified using default: #{entry_point}")
+  if service.type == "ServiceAnsiblePlaybook"
+    entry_point = '/Service/Generic/StateMachines/GenericLifecycle/retire'
+  else
+    entry_point = '/Service/Retirement/StateMachines/ServiceRetirement/Default'
+  end
```
Comment (gmcculloug) [inline content/automate/ManageIQ/Service/Retirement/StateMachines/Methods.class/__methods__/get_retirement_entrypoint.rb]: Rubocop suggestion is to change to this format:
```ruby
entry_point = if service.type == "ServiceAnsiblePlaybook"
                '/Service/Generic/StateMachines/GenericLifecycle/retire'
              else
                '/Service/Retirement/StateMachines/ServiceRetirement/Default'
              end
```

**Comment by billfitzgerald0120** [comment]
@tinaafitz 
Please Review

**Comment by tinaafitz** [comment]
@billfitzgerald0120 Looks good. Could we add a test for this method?

**Comment by gmcculloug** [comment]
Closing in favor of PR https://github.com/ManageIQ/manageiq-content/pull/72

**Pull request closed without merging**